src/Cell.tsx

```diff
@@ -65,16 +68,35 @@ function Cell<R, SR>({
 
   function getCellContent() {
     if (selectedCellProps && selectedCellProps.mode === 'EDIT') {
-      const { editorPortalTarget, ...editorProps } = selectedCellProps.editorContainerProps;
+      const { editorPortalTarget, editorContainerProps, editor2Props } = selectedCellProps;
       const { scrollTop: docTop, scrollLeft: docLeft } = document.scrollingElement || document.documentElement;
       const { left, top } = cellRef.current!.getBoundingClientRect();
       const gridLeft = left + docLeft;
       const gridTop = top + docTop;
 
+      if (column.editor2 !== undefined) {
+        const editor2 = (
+          <column.editor2
+            {...editor2Props}
+            rowIdx={rowIdx}
+            row={row}
+            column={column}
+            left={gridLeft}
```

Comment:
Editor can choose not to use `left, right` props. The only way to override them is in the editor itself. We control `editor2Props` and not the user

Reply:
If the cell is at the right bottom corner, it might end up with weird issue if the editor is customized width and height. That's why I think shall somehow make this left / top controllable by user.